### DISCUSSION TITLE
[2023-04-17] fsic-9th-meeting

### DISCUSSION THREAD

**Discussion by kevinjantw** [bol-edu/caravel-soc#34] [General]
[fsic-9th-meeting 4-17-2023.pdf](https://github.com/bol-edu/caravel-soc/files/11308221/fsic-9th-meeting.4-17-2023.pdf)

**Comment by TonyHo722** [comment]
add link for video 全端 IC 設計工程師伴學松 第九次 https://www.youtube.com/watch?v=bk8kszhKavE
